content/release-10-15-0/announcements-10-15-0.md

```diff
@@ -86,6 +92,75 @@ As announced in [release 10.14](/release-10-14-0/announcements-10-14-0/), the `f
+For a complete description of the new concurrency behavior and options to control it, 
+see [Optimizing requests to Cumulocity IoT with concurrent connections](https://documentation.softwareag.com/pam/10.15.0/en/webhelp/pam-webhelp/index.html#page/pam-webhelp%2Fco-ConApaAppToExtCom_cumulocity_optimizing_requests_to_cumulocity_iot_with_concurrent_connections.html) 
+in the Apama documentation.
+
+The Cumulocity IoT transport now supports multi-tenant deployment. For this purpose, the following new configuration options are now available:
+
+- `CUMULOCITY_MULTI_TENANT_APPLICATION`
+- `CUMULOCITY_MULTI_TENANT_MICROSERVICE_NAME`
+
+See also [Configuring the Cumulocity IoT transport](https://documentation.softwareag.com/pam/10.15.0/en/webhelp/pam-webhelp/index.html#page/pam-webhelp%2Fco-ConApaAppToExtCom_cumulocity_configuring_the_cumulocity_transport.html) 
+and [Working with multi-tenant deployments](https://documentation.softwareag.com/pam/10.15.0/en/webhelp/pam-webhelp/index.html#page/pam-webhelp%2Fco-ConApaAppToExtCom_cumulocity_working_with_multi_tenant_deployments.html)
+in the Apama documentation.
+
+The following new configuration properties, which correspond to existing configuration options in the YAML configuration file, are now available:
+
+- `CUMULOCITY_REQUEST_ALL_DEVICES` (corresponds to `requestAllDevices`)
+- `CUMULOCITY_SUBSCRIBE_ALL_MEASUREMENTS` (corresponds to `subscribeToAllMeasurements`)
+- `CUMULOCITY_SUBSCRIBE_DEVICES` (corresponds to `subscribeToDevices`)
+- `CUMULOCITY_SUBSCRIBE_OPERATIONS` (corresponds to `subscribeToOperations`)
+
+It is recommended to use the new configuration properties instead of editing the YAML configuration file. 
+See also [Configuring the Cumulocity IoT transport](https://documentation.softwareag.com/pam/10.15.0/en/webhelp/pam-webhelp/index.html#page/pam-webhelp%2Fco-ConApaAppToExtCom_cumulocity_configuring_the_cumulocity_transport.html) 
+in the Apama documentation.
+
+##### Docker images
+
+Apama 10.15.0 introduces several new container images provided via Docker Hub and some of the existing container images have changed content.
+See also [Published Apama container images](https://documentation.softwareag.com/pam/10.15.0/en/webhelp/pam-webhelp/index.html#page/pam-webhelp%2Fco-DepAndManApaApp_published_apama_container_images.html)
+in the Apama documentation.
+
+When building images for use as a Cumulocity IoT microservice, then you must use the 
+[softwareag/apama-cumulocity-jre](https://hub.docker.com/r/softwareag/apama-cumulocity-jre) image with the new
+[softwareag/apama-cumulocity-builder](https://hub.docker.com/r/softwareag/apama-cumulocity-builder) image as a builder image. 
+To do this with the default project Dockerfile created by Software AG Designer, either change the `FROM` lines in the Dockerfile appropriately or build using the following flags:
+
+```
+--build-arg APAMA_BUILDER=softwareag/apama-cumulocity-builder:10.15 --build-arg APAMA_IMAGE=softwareag/apama-cumulocity-jre:10.15
```

Comment:
Was this information also added to https://github.com/SoftwareAG/c8y-docs/blob/develop/content/apama/analytics-introduction-bundle/deploying-apps.md#to-deploy-an-apama-application-as-a-microservice ?

Reply:
Yes ;-)